compiler/src/iree/compiler/Dialect/Flow/Transforms/RegionOpUtils.cpp

```diff
@@ -580,6 +580,13 @@ bool isDequantizationLikeOp(Operation *op) {
       outputElementType.getIntOrFloatBitWidth()) {
     return false;
   }
+
+  // Check if there are any operations from math dialect.
+  for (Operation &op : *genericOp.getBody()) {
+    if (op.getDialect() == op.getContext()->getLoadedDialect("math")) {
+      return false;
+    }
+  }
```

Comment:
What happens to this part? I don't see any tests about it.

Reply:
I can add a test.. basically such operations are cloned, so we dont want math operations cloned.

Reply:
Done!

Reply:
Maybe, but we need to see how this evolves. That might be what we need to do.

Reply:
I don't quite follow why we don't want to clone math ops. Is it because they are expensive? Then we might need a better heuristic for things like `math.absi`/`math.ceil`?

Reply:
Yes. I dont know what all will appear in what is known as a dequantization operation. I dont want to let things through in a vaccum. I think it would be better to evolve this on a need-basis.